app/controllers/kms/application_controller.rb

```diff
@@ -1,11 +1,13 @@
 module Kms
   class ApplicationController < ActionController::Base
+    layout 'kms/devise'
```

Comment:
It has no problems with layout, do you think this is need refactoring?

https://github.com/webgradus/kms/blob/master/lib/kms/engine.rb#L14

Used this way: https://github.com/plataformatec/devise/wiki/How-To%3A-Create-custom-layouts#application--devise-config